src/lib/core/questions.ts

```diff
@@ -957,6 +978,7 @@ export interface Answers {
 		| "devcontainer"
 	)[];
 	ecmaVersion?: 2015 | 2016 | 2017 | 2018 | 2019 | 2020;
+	nodeVersion?: "12" | "14" | "16";
```

Comment:
Migration allows 18, but we don't support it here?

Reply:
Oops, I removed 18 after adding it at first because there are no @types/node packages for it yet.